activerecord/lib/arel/visitors/dot.rb

```diff
@@ -148,6 +148,7 @@ def visit_Arel_Nodes_SelectStatement(o)
 
         def visit_Arel_Nodes_UpdateStatement(o)
           visit_edge o, "relation"
+          visit_edge o, "from"
```

Comment:
What is it for?

Reply:
I think the dot formatter is for visualizing the Arel tree as a Graphviz visualization. This adds the new "subtree" to that graph.